Source/Core/DolphinQt/Settings/AudioPane.cpp

```diff
@@ -386,68 +232,100 @@ void AudioPane::OnEmulationStateChanged(bool running)
       !m_dsp_hle->isChecked())
   {
     m_dolby_pro_logic->setEnabled(!running);
-    EnableDolbyQualityWidgets(!running && m_dolby_pro_logic->isChecked());
+    m_dolby_quality_label->setEnabled(!running && m_dolby_pro_logic->isChecked());
+    m_dolby_quality_combo->setEnabled(!running && m_dolby_pro_logic->isChecked());
   }
   if (m_latency_control_supported &&
       AudioCommon::SupportsLatencyControl(Config::Get(Config::MAIN_AUDIO_BACKEND)))
   {
     m_latency_label->setEnabled(!running);
-    m_latency_spin->setEnabled(!running);
+    m_latency_slider->setEnabled(!running);
   }
 
 #ifdef _WIN32
   m_wasapi_device_combo->setEnabled(!running);
 #endif
 }
 
-void AudioPane::OnVolumeChanged(int volume)
-{
-  m_volume_slider->setValue(volume);
-  m_volume_indicator->setText(tr("%1%").arg(volume));
-}
-
 void AudioPane::CheckNeedForLatencyControl()
 {
   std::vector<std::string> backends = AudioCommon::GetSoundBackends();
   m_latency_control_supported =
       std::any_of(backends.cbegin(), backends.cend(), AudioCommon::SupportsLatencyControl);
 }
 
-QString AudioPane::GetDPL2QualityLabel(AudioCommon::DPL2Quality value) const
+void AudioPane::AddDescriptions()
 {
-  switch (value)
-  {
-  case AudioCommon::DPL2Quality::Lowest:
-    return tr("Lowest");
-  case AudioCommon::DPL2Quality::Low:
-    return tr("Low");
-  case AudioCommon::DPL2Quality::Highest:
-    return tr("Highest");
-  default:
-    return tr("High");
-  }
-}
+  static const char TR_HLE_DESCRIPTION[] = QT_TR_NOOP(
+      "High Level Emulation of the DSP. Fast, but not always accurate, also lacks Dolby Pro Logic "
+      "II decoding.<br><br><dolphin_emphasis>If unsure, select this mode.</dolphin_emphasis>");
+  static const char TR_LLE_DESCRIPTION[] =
+      QT_TR_NOOP("Low Level Emulation of the DSP, via a recompiler. Slower, but more accurate, "
+                 "also enables Dolby Pro Logic II decoding on certain audio backends.");
+  static const char TR_INTERPRETER_DESCRIPTION[] =
+      QT_TR_NOOP("Low Level Emulation of the DSP, via a interpreter. Slowest, for debugging "
+                 "purposes only.<br><br><dolphin_emphasis>Not recommended, consider using DSP HLE "
+                 "or DSP LLE Recompiler instead.</dolphin_emphasis>");
```

Comment:
Small typo I introduced in my previous review, edited the comment but it apparently didn't went through (via a interpreter => via an interpreter): 

`Low Level Emulation of the DSP, via an interpreter. Slowest, for debugging purposes only.<br><br><dolphin_emphasis>Not recommended, consider using DSP HLE or DSP LLE Recompiler instead.</dolphin_emphasis>`